src/snowflake/snowpark/dataframe_stat_functions.py

```diff
@@ -92,14 +92,8 @@ def approx_quantile(
         if _emit_ast:
             # Add an assign node that applies SpDataframeStatsApproxQuantile() to the input, followed by its Eval.
```

Comment:
the constructor here prob. needs to be modified to remove self._ast_stmt (+ its parameter)